index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+    <!-- <script defer src="./src/scripts/index.js"></script> -->
+    <link rel="stylesheet" href="./dist/css/index.css">
+    <title>Document</title>
+</head>
+<body class="bg-black" style="height: 100vh;">
+  <div class="container min-vh-100 d-flex justify-content-center align-items-center">
+    <div class="row vw-100">
+      <div class="col-md-6 mb-5 d-flex align-content-center justify-content-center">
+        <img src="./public/brand-logo.svg" alt="Brand Logo" />
+      </div>
+      <div class="col-md-6">
+        <h3 class="text-white">Welcome Back</h3>
+        <form action="/profile.html" class="w-100 d-flex flex-column">
+          <div class="form-floating mb-3">
+            <input type="email" class="form-control" placeholder="Type your username" id="email" required>
+            <label for="email" class="form-label">
+              <i class="bi bi-person"></i>
+              Username
```

Comment:
Hey you should change Username to Email when type="email", so it's clear that users need to use an email and not a generic username

Reply:
Good point. I've changed the label from username to email.